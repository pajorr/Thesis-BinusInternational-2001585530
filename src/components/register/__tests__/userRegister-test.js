import React from 'react';
import { shallow } from 'enzyme';

import Register from '../userRegister';


describe('userLogin', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Register debug/>);

        expect(component).toMatchSnapshot();
    });
});
