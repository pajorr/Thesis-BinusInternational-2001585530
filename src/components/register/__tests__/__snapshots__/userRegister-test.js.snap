// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userLogin should render correctly in "debug" mode 1`] = `
<WithStyles(ForwardRef(Container))
  component="main"
  maxWidth="xs"
>
  <CssBaseline />
  <div>
    <WithStyles(ForwardRef(Typography))
      component="h1"
      variant="h5"
    >
      Sign up
    </WithStyles(ForwardRef(Typography))>
    <form
      noValidate={true}
    >
      <WithStyles(ForwardRef(Grid))
        container={true}
        spacing={2}
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="fname"
            autoFocus={true}
            fullWidth={true}
            label="Name"
            name="name"
            onChange={[Function]}
            required={true}
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="email"
            fullWidth={true}
            label="Email Address"
            name="email"
            onChange={[Function]}
            required={true}
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <WithStyles(ForwardRef(TextField))
            autoComplete="current-password"
            fullWidth={true}
            label="Password"
            name="password"
            onChange={[Function]}
            required={true}
            type="password"
            variant="outlined"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <Link
            to="/login"
          >
            <WithStyles(ForwardRef(Button))
              fullWidth={true}
              onClick={[Function]}
              type="button"
              variant="contained"
            >
              Sign Up
            </WithStyles(ForwardRef(Button))>
          </Link>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          container={true}
          justify="flex-end"
        >
          <WithStyles(ForwardRef(Grid))
            item={true}
          >
            <Link
              href="#"
              to="/login"
              variant="body2"
            >
              Already have an account? Sign in
            </Link>
          </WithStyles(ForwardRef(Grid))>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </form>
  </div>
</WithStyles(ForwardRef(Container))>
`;
